.github/workflows/fastapi.yml

```diff
@@ -0,0 +1,40 @@
+name: fastapi-testsuite
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+  fastapi-test:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: fastapi
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
```

Comment:
```suggestion
        python-version: [3.6, 3.7, 3.8, 3.9]
```

Reply:
Just curious, @Kyle-Verhoog @Genegrady is it worth testing the fastapi tests on 4 different python versions?